tensorflow/compiler/xla/service/hlo_instruction.h

```diff
@@ -1384,6 +1384,14 @@ class HloInstruction {
   }
   Status set_backend_config(const tensorflow::protobuf::Message& proto);
 
+  void set_frontend_attributes(FrontendAttributes frontend_attributes) {
```

Comment:
We'll also need to serialize and parse these.  Are you going to add support for parsing / serializing in subsequent commits?

Reply:
Sorry, I'm not familiar with the Serialization API: I'm going to add the frontend attributes to the `ToProto` method, anything else there is to do ?

Reply:
See hlo_parser.h and hlo_parser.cc and `HloModule::ToString`

Reply:
Done